index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content= "width=device-width, initial-scale=1.0">
+        <meta content="text" charset="utf-8">
+        <link rel="stylesheet" href="styles.css">
+        <title>Kiplaacollo Portfolio website</title>
+
+    </head>
+    <body>
+        <header class="head">
+                <h3 class="logo">Welcome</h3>
+                <img src="images/Enabled.png" alt="menu icon">
+        </header>
+        <div class="content">
+                <div class="primary_txt">
+                    <h2 class="top_color">Hey there. I'm Kiplaacollo </h2>
+                    <h2 class="bottom_color">I'm a software developer </h2>
+                </div>
+                <div class="supportive_txt">
+                    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.</p>
+                </div>
+            <div class="social_icon_container">
+                <a href="#" class="social-icon">
+                    <img src="images/twitter.png" alt="twiter icon">
+               </a>
+               <a href="#" class="social-icon">
+                   <img src="images/Linkedin icon.png" alt="Linkedin icon">
+               </a>
+               <a href="#" class="social-icon">
+                   <img src="images/medium.png" alt="medium icon">
+               </a>
+               <a href="#" class="social-icon">
+                   <img src="images/github.png" alt="github icon">
+               </a>
+               <a href="#" class="social-icon">
+                   <img src="images/angellist.png" alt="angellist icon">
+               </a>
+            </div>
+            
+        </div>
```

Comment:
- [ ] Kindly I recommend you to use `section` tag for headline section instead of `div` tag because It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

Reply:
Thanks. At first, I used sections for different sections, and did not look good. Let me change it.